File: t3/viabnb.tier3.dataserver.soapws/src/main/java/dk/viabnb/sep3/group6/dataserver/rest/t3/models/HostRegistrationRequest.java
package dk.viabnb.sep3.group6.dataserver.rest.t3.models;

public class HostRegistrationRequest
{
  private int id;
  private String personalImage;
  private String cprNumber;
  private RequestStatus status = RequestStatus.NotAnswered;

  public HostRegistrationRequest(int id, String personalImage, String cprNumber,
      RequestStatus status)
  {
    this.id = id;
    this.personalImage = personalImage;
    this.cprNumber = cprNumber;
    this.status = status;
  }

  public int getId()
  {
    return id;
  }

  public void setId(int id)
  {
    this.id = id;
  }

  public String getPersonalImage()
  {
    return personalImage;
  }

  public void setPersonalImage(String personalImage)
  {
    this.personalImage = personalImage;
  }

  public String getCprNumber()
  {
    return cprNumber;
  }

  public void setCprNumber(String cprNumber)
  {
    this.cprNumber = cprNumber;
  }

  public RequestStatus getStatus()
  {
    return status;
  }

  public void setStatus(RequestStatus status)
  {
    this.status = status;
  }
}

File: t1/SEP3BlazorT1Client/SEP3BlazorT1Client/Models/HostRegistrationRequest.cs
using System.ComponentModel.DataAnnotations;

namespace SEP3BlazorT1Client.Models
{
    public class HostRegistrationRequest
    {
        [Required]
        public int Id { get; set; }
        [Required]
        public string PersonalImage { get; set; }
        [Required]
        public string CprNumber { get; set; }
        public Host Host { get; set; }

        [Required] public RequestStatus Status { get; set; } = RequestStatus.NotAnswered;
    }
}